rock-unit-rank.ttl

```diff
@@ -0,0 +1,861 @@
+  rdfs:isDefinedBy <https://linked.data.gov.au/def/rock-unit-rank> ;
+  skos:definition "An intrusion sensu lato that is kilometre-scale or larger, and cylindrical, lenticular or tabular (Gillespie & Leslie, 2021)."@en ;
+  skos:inScheme <https://linked.data.gov.au/def/rock-unit-rank> ;
+  skos:notation "rrplu" ;
+  skos:prefLabel "pluton"@en ;
+  skos:related geof:IgneousIntrusiveRockUnit ;
+  dcterms:provenance "Gillespie & Leslie, 2021"@en ;
+.
+rur:PlutonCluster
+  a skos:Concept ;
+  dcterms:references "Jell, P.A. (Editor), 2013: Geology of Queensland. Geological Survey of Queensland, Brisbane, 1063 pages. <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/reports-news/geology-queensland-book-map>"@en ;
+  rdfs:isDefinedBy <https://linked.data.gov.au/def/rock-unit-rank> ;
+  skos:definition "A cluster of related plutons [e.g., the Forsayth Batholith (of the Forsayth Supersuite, Forsayth Subprovince, Etheridge Province, northern Queensland)], although the term ‘pluton cluster’ has not been applied per se to the unit, although it ‘comprises ~30 individual plutons or clusters of plutons 2–10 km in diameter’ (Jell, 2013, pp. 61, 80–81)."@en ;
+  skos:inScheme <https://linked.data.gov.au/def/rock-unit-rank> ;
+  skos:notation "rrpcl" ;
+  skos:prefLabel "pluton cluster"@en ;
+  skos:related geof:IgneousIntrusiveRockUnit ;
+  dcterms:provenance "Jell, 2013, pp. 61, 80–81"@en ;
+.
+rur:Ring-dyke
+  a skos:Concept ;
+  dcterms:references "Gillespie, M.R. & Leslie, A.G., 2021: BRUCS: a new system for classifying and naming mappable rock units. Journal of the Geological Society, 178(4), jgs2020-212. <https://jgs.lyellcollection.org/content/jgs/178/4/jgs2020-212.full.pdf>"@en ;
+  rdfs:isDefinedBy <https://linked.data.gov.au/def/rock-unit-rank> ;
+  skos:altLabel "Ring-dike"@en ;
+  skos:definition "A sheet that is arcuate or annular in plan, and usually vertical or inclined steeply outwards (Gillespie & Leslie, 2021)."@en ;
+  skos:inScheme <https://linked.data.gov.au/def/rock-unit-rank> ;
+  skos:notation "rrrdy" ;
+  skos:prefLabel "ring-dyke"@en ;
+  skos:related geof:IgneousIntrusiveRockUnit ;
+  dcterms:provenance "Gillespie & Leslie, 2021"@en ;
+.
+rur:RingComplex
+  a skos:Concept ;
+  dcterms:references "Gillespie, M.R. & Leslie, A.G., 2021: BRUCS: a new system for classifying and naming mappable rock units. Journal of the Geological Society, 178(4), jgs2020-212. <https://jgs.lyellcollection.org/content/jgs/178/4/jgs2020-212.full.pdf>"@en ;
+  rdfs:isDefinedBy <https://linked.data.gov.au/def/rock-unit-rank> ;
+  skos:broader rur:Complex ;
+  skos:definition "A unit comprising multiple ring-intrusions and/or ring-dykes, cone-sheets, ring-dyke-swarms and cone-sheet-swarms, and their country-rock (Gillespie & Leslie, 2021)."@en ;
+  skos:inScheme <https://linked.data.gov.au/def/rock-unit-rank> ;
+  skos:notation "rrrco" ;
+  skos:prefLabel "ring-complex"@en ;
+  skos:related rur:ComplexRockUnit ;
```

Comment:
```suggestion
  skos:related geof:ComplexRockUnit ;
```

Assume this is geof: as per the other references

Reply:
It’s my proposal to add ComplexRockUnit to geof, along with others, e.g, IntrusiveRockUnit, TectonometamoprphicRockUnit, etc. 

For ComplexRockUnit, we would then have:
geof:ComplexRockUnit a skos:Concept ; 
skos:altLabel "complex unit"@en,
    "composite unit"@en ;
etc.

Other changes to geof include, e.g., 'geof:tectonostratigraphicEntity a skos:Concept ;' being changed to 'geof:TectonostratigraphicRockUnit a skos:Concept ;'

I originally had ComplexRockUnit, IntrusiveRockUnit, etc. in this current rur vocab, but, in the way we have constructed geof, it belongs there, with the other types of rock units (lithostratigraphic, tectonostratigraphic, etc.), unless we undergo a major reconstruct.

Reply:
That's fine. You just need to make sure the rur: and geof: prefixes are pointing to the right place, e.g. change rur:ComplexRockUnit to geof:ComplexRockUnit